Requirements for SSE.md

```diff
@@ -4,15 +4,15 @@
 
 ### Case list
 
-#### TODO: Usecase 1 Link
+#### [Usecase 1](#use-case-3)
```

Comment:
I made these changes to, hopefully github has a merge conflict resolver for us to pick which lines we want the master to have. Besides that everything else looks good.

Reply:
Awesome sounds good! Im sure if worse case scenarios if they are just HREF links we can just edit them and clean it up tomorrow in the Master branch.